v2/slogger/rolling_file_appender/rotation_time.go

```diff
@@ -31,7 +31,7 @@ func (self RotationTimeSlice) Swap(i, j int) {
 	self[i], self[j] = self[j], self[i]
 }
 
-var rotatedTimeRegExp = regexp.MustCompile(`\.(\d+-\d\d-\d\dT\d\d-\d\d-\d\d)(-(\d+))?$`)
+var rotatedTimeRegExp = regexp.MustCompile(`\.(\d+-\d\d-\d\dT\d\d-\d\d-\d\d)(-(\d+))?(.gz)?$`)
```

Comment:
The period is not escaped.

So just adding (.gz) can match:

agz
fgz
zgz

etc.....

Reply:
Fixed, thanks!